INSTALL.md

```diff
@@ -69,6 +69,24 @@ at [localhost:1081](http://localhost:1081).
 - Keep an eye on console in case of compiling errors.
 - [Read more](https://github.com/Trustroots/trustroots/wiki/Development)
 
+## Enable collecting statistics
+
+* 1. [Install InfluxDB](https://docs.influxdata.com/influxdb/latest/introduction/installation/) 0.9+ and run it (type `influxd`) at default port (8086)
```

Comment:
@mrkvon good question.

Regarding logging
- Could you add to [app init logging](https://github.com/Trustroots/trustroots/blob/master/config/lib/app.js#L37-L45) a line telling if InfluxDB is configured?
- If it's not configured, it should probably `console.error` only on `production` env? On test/development it can fail silently. Not a big deal to have a few console.log warnings appearing now, but once every feature sends out a bunch of points, it'll be too much.

About monitoring
- There will be a separate monitoring on some graphs (e.g. signups, so if those drop, we know that either signup broke or collecting stats broke and monitoring script will notify us).
- Service monitoring/alerting shouldn't be handled via app anyway, it's done via separate system.